sys/linux/bpf.txt

```diff
@@ -39,6 +39,7 @@ bpf$OBJ_GET_PROG(cmd const[BPF_OBJ_GET], arg ptr[in, bpf_obj_get], size len[arg]
 bpf$BPF_PROG_ATTACH(cmd const[BPF_PROG_ATTACH], arg ptr[in, bpf_attach_arg], size len[arg])
 bpf$BPF_PROG_DETACH(cmd const[BPF_PROG_DETACH], arg ptr[in, bpf_detach_arg], size len[arg])
 bpf$BPF_PROG_TEST_RUN(cmd const[BPF_PROG_TEST_RUN], arg ptr[in, bpf_test_prog_arg], size len[arg])
+bpf$BPF_PROG_RUN(cmd const[BPF_PROG_TEST_RUN], arg ptr[in, bpf_test_prog_arg], size len[arg])
```

Comment:
I don't think we need this. It's the same syscall we already have.
If you want coverage of all enum values for grepping purposes, maybe just add a comment saying that BPF_PROG_RUN is the same as BPF_PROG_TEST_RUN?

Reply:
Oh, of course :facepalm: I think I added it without thinking :-/

I removed that commit.